tests/importer/spreadsheet/test_ingest_xlsimporter.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+
+from mock import MagicMock
+
+import ingest
+from ingest.api.ingestapi import IngestApi
+from ingest.importer.importer import XlsImporter
+
+import ingest.importer.submission
+from ingest.utils.SubmissionError import ImporterError
+
+
+class IngestXlsImporterTest(TestCase):
```

Comment:
I think this should be in `test_importer.py`.

Reply:
Those tests are all about workbook and worksheet tests which seem irrelevant to my tests.

Reply:
Those are tests for `WorksheetImporter` and `WorkbookImporter` which are both in `importer.py`. The `importer.py` module currently contains all components related to excel spreadsheet processing. We can restructure this (and we probably should as it's growing), but if it stays this way, it's easier to think that the tests for `importer.py` are all in `test_importer.py`.